agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml

```diff
@@ -18,6 +18,12 @@ services:
             APP_PORT: 3000
             AZURE_STORAGE_ACCESS_KEY:
             EVENT_HUB_SAS_KEY: ${EVENT_HUB_SAS_KEY_INPUT}
+            #kafka 
+            KAFKA_TOPIC_INPUT: ${KAFKA_TOPIC_INPUT}
```

Comment:
Btw, you can prevent duplicating all the config using yaml anchors. Here's an example:

```diff
diff --git a/agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml b/agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml
index a722eda..08eab18 100644
--- a/agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml
+++ b/agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml
@@ -1,5 +1,19 @@
 version: "3.7"
 
+x-shared-environment:
+    &shared-environment
+    AZURE_STORAGE_ACCESS_KEY: ${AZURE_STORAGE_ACCESS_KEY}
+    AZURE_STORAGE_ACCOUNT: ${AZURE_STORAGE_ACCOUNT}
+    EVENTHUB_KAFKA_CONNECTION_STRING: ${EVENTHUB_KAFKA_CONNECTION_STRING}
+    EVENT_HUB_NAME: ${EVENT_HUB_NAME_INPUT}
+    EVENT_HUB_NAMESPACE: ${EVENT_HUB_NAMESPACE}
+    EVENT_HUB_SAS_KEY: ${EVENT_HUB_SAS_KEY_INPUT}
+    EVENT_HUB_SAS_POLICY: ${EVENT_HUB_SAS_POLICY}
+    KAFKA_ADDRESS: ${KAFKA_ADDRESS}
+    KAFKA_TIMEOUT: ${KAFKA_TIMEOUT}
+    LEASE_CONTAINER_NAME: ${LEASE_CONTAINER_NAME_INPUT}
+    MESSAGING_TYPE: ${MESSAGING_TYPE}
+
 services:
     input-reader:
         image: ${CONTAINER_REG}/input_reader:${TAG}
@@ -8,22 +22,11 @@ services:
                 aliases:
                     - input
         environment:
-            MESSAGING_TYPE: ${MESSAGING_TYPE}
-            AZURE_STORAGE_ACCOUNT: ${AZURE_STORAGE_ACCOUNT}
-            EVENT_HUB_NAMESPACE: ${EVENT_HUB_NAMESPACE}
-            EVENT_HUB_NAME: ${EVENT_HUB_NAME_INPUT}
-            EVENT_HUB_SAS_POLICY: ${EVENT_HUB_SAS_POLICY}
-            LEASE_CONTAINER_NAME: ${LEASE_CONTAINER_NAME_INPUT}
+            << : *shared-environment
             APP_HOST: sample
             APP_PORT: 3000
-            AZURE_STORAGE_ACCESS_KEY:
-            EVENT_HUB_SAS_KEY: ${EVENT_HUB_SAS_KEY_INPUT}
-            #kafka 
             KAFKA_TOPIC_INPUT: ${KAFKA_TOPIC_INPUT}
-            KAFKA_TIMEOUT: ${KAFKA_TIMEOUT}
             KAFKA_CONSUMER_GROUP: ${KAFKA_CONSUMER_GROUP}
-            KAFKA_ADDRESS: ${KAFKA_ADDRESS}
-            EVENTHUB_KAFKA_CONNECTION_STRING: ${EVENTHUB_KAFKA_CONNECTION_STRING}
 
     output-writer:
         image: ${CONTAINER_REG}/output_writer:${TAG}
@@ -34,21 +37,10 @@ services:
                 aliases:
                     - output
         environment:
-            MESSAGING_TYPE: ${MESSAGING_TYPE}
-            AZURE_STORAGE_ACCOUNT: ${AZURE_STORAGE_ACCOUNT}
-            AZURE_STORAGE_ACCESS_KEY: ${AZURE_STORAGE_ACCESS_KEY}
-            LEASE_CONTAINER_NAME: ${LEASE_CONTAINER_NAME_OUTPUT}
-            EVENT_HUB_NAMESPACE: ${EVENT_HUB_NAMESPACE}
-            EVENT_HUB_NAME: ${EVENT_HUB_NAME_OUTPUT}
-            EVENT_HUB_SAS_POLICY: ${EVENT_HUB_SAS_POLICY}
+            << : *shared-environment
             OUTPUT_WRITER_PORT: 6000
             OUTPUT_WRITER_HOST: output
-            EVENT_HUB_SAS_KEY: ${EVENT_HUB_SAS_KEY_OUTPUT}
-            #kafka             
             KAFKA_TOPIC_OUTPUT: ${KAFKA_TOPIC_OUTPUT}
-            KAFKA_TIMEOUT: ${KAFKA_TIMEOUT}
-            KAFKA_ADDRESS: ${KAFKA_ADDRESS}
-            EVENTHUB_KAFKA_CONNECTION_STRING: ${EVENTHUB_KAFKA_CONNECTION_STRING}

```

Reply:
Just added this -- thanks!